generator/.gitignore

```diff
@@ -0,0 +1 @@
+sigs.yaml
```

Comment:
With the changes, I think this should be `generated/` to ignore that whole directory.